released/learning_journey/btp200/README.md

```diff
@@ -0,0 +1,64 @@
+# Learning Journey BTP200
+## Overview
+
+This folder contains the files for the Terraform exercise of the SAP Learing Journey BTP200
+
+## Content of setup
+
+The setup comprises the following resources:
+
+- Creation of an SAP BTP subaccount or using an existing subaccount
+- Entitlements of services
+   * SAP Business Application Studio
+   * SAP Build Work Zone, standard edition
+   * Continous Integration & Delivery - optional
+  
+- Subscriptions of the services
+- Role collection assignments to users
+
+## Deploying the resources
+
+Make sure that you are familiar with SAP BTP and know both the [Get Started with btp-terraform-samples](https://github.com/SAP-samples/btp-terraform-samples/blob/main/GET_STARTED.md) and the [Get Started with the Terraform Provider for BTP](https://developers.sap.com/tutorials/btp-terraform-get-started.html)
+
+To deploy the resources you must:
+
+1. Create a file `secret.auto.tfvars` and maintain the credentials for the BTP provider
+
+   ```hcl
+   btp_user = "<Email address of your BTP user>"
+   btp_password = "<Password of your BTP user>"
+   ```
+   as an alternative you can set this credentials also as environment variables
+   
+   ```bash
+   export btp_user ='<Email address of your BTP user>'
```

Comment:
I think these Variables need to be in capital letters namely BTP_USERNAME and BTP_PASSWORD